spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/DataFlowTemplate.java

```diff
@@ -51,62 +51,67 @@
 	/**
 	 * REST client for stream operations.
 	 */
-	private final StreamOperations streamOperations;
```

Comment:
Why remove `final`? What happens if none of the feature flags are enabled? Seems those are left `null`, so why not make that an explicit `else` branch (if that's really what expected downstream)?